packages/palette/src/svgs/ArrowDownCircleFillIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import { color } from "../helpers"
+import { Icon, IconProps, Path, Title } from "./Icon"
+
+/** ArrowDownCircleFillIcon */
+export const ArrowDownCircleFillIcon: React.SFC<IconProps> = ({
+  title = "Down",
+  ...props
+}) => {
+  return (
+    <Icon {...props} viewBox="0 0 18 18">
+      <Title>{title}</Title>
+      <Path
+        d="M16 9.5C16 13.366 12.866 16.5 9 16.5C5.13401 16.5 2 13.366 2 9.5C2 5.63401 5.13401 2.5 9 2.5C12.866 2.5 16 5.63401 16 9.5Z"
+        fill={color(props.fill)}
+      />
+      <Path d="M9 12.7L6 7.5H12L9 12.7Z" fill="white" />
```

Comment:
These fills should actually be holes. It'd be better if we could "flatten" the SVG to boolean out the shape from the other paths? (Consider if we were to support a dark mode)

Reply:
@willanderson Could you make an SVG file that has a hole, rather than a path that overrides the other `<path>`?

Reply:
If you pass the raw svg through [svgo](https://jakearchibald.github.io/svgomg/) it might do that for you.

Reply:
Sadly svgo couldn't merge paths properly (it still left the two paths that overrides the circle with a white background).